python/cudnn/sparse_attention/__init__.py

```
@@ -0,0 +1,37 @@
+    symbol = getattr(module, symbol_name)
+    globals()[name] = symbol
+    return symbol
+
+
+def __getattr__(name):
+    if name in _SYMBOLS:
+        return _load_symbol(name)
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
+
+__all__ = [*_SYMBOLS.keys()]
```

Ruff (0.16.2)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)